jac-uk/apply
# Feat/1136 align ethnicity fields with census categories

## What's included?
Closes #1136 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the preview link: https://jac-apply-develop--pr1146-feat-1136-align-ethn-pnfr9snl.web.app/vacancy/b96p6e9GTAkVUhbggHom/
- Sign in with one of user accounts:
```
application-0001@jac-dummy-email.jac
application-0002@jac-dummy-email.jac
application-0003@jac-dummy-email.jac
application-0004@jac-dummy-email.jac
application-0005@jac-dummy-email.jac
application-0006@jac-dummy-email.jac
application-0007@jac-dummy-email.jac
application-0008@jac-dummy-email.jac
application-0009@jac-dummy-email.jac

# password
1qazZAQ!1qaz
```
- Go to the `Equality and diversity` section
- `Roma`, `Arab` should appear as option in the selection of ethnic group 
- `Prefer not to say` should appear in its own section in the selection of ethnic group
- For testing the new ethnic options will only appear in exercises with start date after `2024-03-01`( after passing the test, the date will change back to `2024-04-01` for production environment )
<img width="660" alt="Screenshot 2024-03-06 at 16 41 04" src="https://github.com/jac-uk/apply/assets/156695133/59a37a86-9c70-44e7-800d-8ac238f0b004">

- Go to preview link: https://jac-apply-develop--pr1146-feat-1136-align-ethn-pnfr9snl.web.app/apply/pmhY2nNEQFg4OXzSDPYw
- Go to the `Equality and diversity` section
- `Roma`, `Arab` should `not` appear as option in the selection of ethnic group 
- For testing the new ethnic options will only appear in exercises with start date after `2024-03-01`( after passing the test, the date will change back to `2024-04-01` for production environment )

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
